ui/nuxeo-i18n-behavior.js

```diff
@@ -139,4 +139,18 @@ export const I18nBehavior = {
     // this.i18n = null;
     this.set('i18n', window.nuxeo.I18n.translate);
   },
+
+  i18nWithPrefix(prefix, key, ...args) {
```

Comment:
Probably you may want to take a look to `nuxeo-format-behavior` since there are similar methods being done there. For coherency purposes, we may want to have a `formatUserPermission` method.
e.g. https://github.com/nuxeo/nuxeo-elements/blob/master/ui/nuxeo-format-behavior.js#L133-L143